src/main/java/com/wolox/training/models/User.java

```diff
@@ -0,0 +1,123 @@
+package com.wolox.training.models;
+
+import com.wolox.training.exception.BookAlreadyOwnedException;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * This class represent the User entity
+ */
+@Entity(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(nullable = false)
+    @NotNull
+    private String username;
+
+    @Column(nullable = false)
+    @NotNull
+    private String name;
+
+    @Column(nullable = false)
+    @NotNull
+    private LocalDate birthDate;
+
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(
+            name = "user_book",
+            joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+            inverseJoinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id"),
+            uniqueConstraints = @UniqueConstraint(columnNames = {"user_id", "book_id"}))
```

Comment:
`@JoinTable` is not necessary, JPA does this automatically

Reply:
I have added the "unique constraints" inside the @JoinTable annotation to create a composite unique key, if I remove the @JoinTable I would lose this

Reply:
you're right, great job